sealer/cmd/delete.go

```diff
@@ -52,20 +81,35 @@ var deleteCmd = &cobra.Command{
 				}
 			}
 		}
-		applier, err := apply.NewApplierFromFile(clusterFile)
-		if err != nil {
-			logger.Error(err)
-			os.Exit(1)
-		}
-		if err = applier.Delete(); err != nil {
-			logger.Error(err)
-			os.Exit(1)
+		if deleteArgs.Nodes != "" || deleteArgs.Masters != "" {
+			applier := apply.NewScalingApplierFromArgs(deleteClusterFile, deleteArgs, false)
+			if applier == nil {
+				os.Exit(1)
+			}
+			if err := applier.Apply(); err != nil {
+				logger.Error(err)
+				os.Exit(1)
+			}
```

Comment:
return fast.

```
if xxx {
    xxx
    return
}
xxxx
```

when the `else` is end of the function, we can return in `if`